cluster/manifests/aws-node-decommissioner/01-rbac.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: "aws-node-decommissioner"
+  namespace: "default"
```

Comment:
This should be deployed to `kube-system` since it's a system component.

Reply:
So all components in the manifests folder are `kube-system`?